Build on intro context (#8)
Partydonk/partydonk: README.md

<img src="Images/Partydonk.png" width="320" alt="Partydonk Logo" />

This project lays down foundational features and changes that we want to bring to .NET to support both deep learning workloads and scientific workloads. These are both growth areas with needs .NET is well placed to service. However, without these changes using .NET is more difficult, so it is not a natural first choice in these domains, and the high level of development friction does not bring joy for developers where .NET is currently their preferred platform. 

> 

Author: ```suggestion

```

We seek to address both these deficiencies for these workloads, making .NET both a natural fit and a delight to use.

> 

Author: ```suggestion
```

> 

Author: ```suggestion
```

<img src="Images/Partydonk.png" width="320" alt="Partydonk Logo" />

> This project lays down foundational features and changes that we want to bring to .NET to support both deep learning workloads and scientific workloads. These are both growth areas with needs .NET is well placed to service. However, without these changes using .NET is more difficult, so it is not a natural first choice in these domains, and the high level of development friction does not bring joy for deveopers where .NET is currently their perferred platform. For these workloads we seek to address both these deficiencies, making .NET both a natural fit and a joy to use.

Author: ```suggestion
This project lays down foundational features and changes that we want to bring to .NET to support both deep learning workloads and scientific workloads. These are both growth areas with needs .NET is well placed to service. However, without these changes using .NET is more difficult, so it is not a natural first choice in these domains, and the high level of development friction does not bring joy for developers where .NET is currently their preferred platform. For these workloads we seek to address both these deficiencies, making .NET both a natural fit and a delight to use.

```

<img src="Images/Partydonk.png" width="320" alt="Partydonk Logo" />

> This project lays down foundational features and changes that we want to bring to .NET to support both deep learning workloads and scientific workloads. These are both growth areas with needs .NET is well placed to service. However, without these changes using .NET is more difficult, so it is not a natural first choice in these domains, and the high level of development friction does not bring joy for developers where .NET is currently their preferred platform. For these workloads we seek to address both these deficiencies, making .NET both a natural fit and a delight to use.

Author: ```suggestion
This project lays down foundational features and changes that we want to bring to .NET to support both deep learning workloads and scientific workloads. These are both growth areas with needs .NET is well placed to service. However, without these changes using .NET is more difficult, so it is not a natural first choice in these domains, and the high level of development friction does not bring joy for developers where .NET is currently their preferred platform. 

We seek to address both these deficiencies for these workloads, making .NET both a natural fit and a delight to use.

```
